jupyterlab-server-proxy/package.json

```diff
@@ -12,7 +12,8 @@
     "url": "https://github.com/jupyterhub/jupyter-server-proxy/issues"
   },
   "license": "BSD-3-Clause",
-  "author": "Yuvi Panda",
+  "author": "Ryan Lovett & Yuvi Panda",
+  "author_email": "rylo@berkeley.edu",
```

Comment:
@yuvipanda @ryanlovett we're unifying the metadata between the python and javascript extensions. Should we use these authors and email, or a generic jupyter one?